app/models/differential_expression_result.rb

```diff
@@ -132,14 +119,24 @@ def set_cluster_name
     self.cluster_name = cluster_group.name
   end
 
-  def has_observed_values?
-    if observed_values.count < MIN_OBSERVED_VALUES
-      errors.add(:observed_values, "must have at least #{MIN_OBSERVED_VALUES} values")
+  def comparisons_available?
```

Comment:
Would this add an error when the user uploads a DE file that contains only 1 one-vs-rest comparison?

Reply:
No - this would throw an error if there was only one label (meaning there's nothing to compare against).